test/base/SynchronizedTest.ooc

```diff
@@ -1,39 +1,39 @@
 use ooc-base
 import threading/Thread, os/Time
 
-TestClass: class {
-	refCounter: ReferenceCounter
-	init: func {
-		refCounter = ReferenceCounter new(this)
+version(debugTests) {
+	TestClass: class {
+		refCounter: ReferenceCounter
+		init: func {
+			refCounter = ReferenceCounter new(this)
+		}
+		dispose: func {
```

Comment:
So you should probably open issues for the other tests you just removed the printouts from, too.